website/pages/docs/commands/connect/envoy.mdx

```diff
@@ -126,12 +126,6 @@ proxy configuration needed.
 - `-deregister-after-critical` - The amount of time the gateway services health check can
   be failing before being deregistered. This flag is used in combination with `-register`
 
--> **Note:** If ACLs are enabled, a token granting `service:write` for the
```

Comment:
I moved this note to the gateway-specific pages.

Maybe we should keep a note in the CLI docs about this though. It would just need to provide guidance for all types of gateways.